cookbook_se/recipes/04_remote_flyte/lp_notifications.py

```diff
@@ -0,0 +1,109 @@
+"""
+
+06: Getting notifications on workflow termination
+-------------------------------------------------
+
+For background on launch plans, refer to :any:`launch_plans`.
+
+Currently notifications are supported on workflow termination and can be of three flavors:
+
+- `Email <#howto-email-notifications>`__
+- `PagerDuty <#howto-pagerduty-notifications>`__
+- `Slack <#howto-slack>`__
+
+You can combine notifications in a single workflow to trigger for different combinations of recipients, domains and terminal phases.
+Notifications can be used in one-off executions or for scheduled workflows when for example, you want to be notified of failure.
+
+Terminal workflow phases include:
+
+- ``WorkflowExecution.Phase.FAILED``
+
+- ``WorkflowExecution.Phase.TIMED_OUT``
+
+- ``WorkflowExecution.Phase.ABORTED``
+
+- ``WorkflowExecution.Phase.SUCCESS``
+
+Notifications can be customized by Flyte platform administrators using templatized variables in the flyteadmin
+`emailer config <https://lyft.github.io/flyte/administrator/install/configure/admin.html#setting-up-workflow-notifications>`_.
+
+"""
+
+# %%
+# Let's consider the following example workflow:
+from flytekit import LaunchPlan, task, workflow
+from flytekit.annotated.notification import Email
+from flytekit.models.core.execution import WorkflowExecutionPhase
+
+
+@task
+def double_int_and_print(a: int) -> str:
+    return str(a * 2)
+
+
+@workflow
+def int_doubler_wf(a: int) -> str:
+    doubled = double_int_and_print(a=a)
+    return doubled
+
+
+int_doubler_wf_lp = LaunchPlan.create(
```

Comment:
comment here to explain that this launch plan is for the case when succeeded?

Reply:
done